charts/bulk-scan-orchestrator/values.preview.template.yaml

```diff
@@ -4,19 +4,27 @@ java:
       secretRef: servicebus-secret-namespace-{{ .Release.Name }}
       key: connectionString
   environment:
-    APPINSIGHTS_INSTRUMENTATIONKEY: "00000000-0000-0000-0000-000000000000"
+    S2S_URL: "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
+    IDAM_API_URL: "https://idam-api.aat.platform.hmcts.net"
+    IDAM_CLIENT_REDIRECT_URI: "https://bulk-scan-orchestrator-aat.service.core-compute-aat.internal/oauth2/callback"
+    CORE_CASE_DATA_API_URL: "http://ccd-data-store-api-aat.service.core-compute-aat.internal"
+    DOCUMENT_MANAGEMENT_URL: "http://dm-store-aat.service.core-compute-aat.internal"
+    TRANSFORMATION_URL_BULKSCAN: "http://bulk-scan-sample-app-aat.service.core-compute-aat.internal"
     ENVELOPES_QUEUE_CONNECTION_STRING: "$(SB_CONN_STRING);EntityPath=envelopes"
     ENVELOPES_QUEUE_MAX_DELIVERY_COUNT: "10"
     PROCESSED_ENVELOPES_QUEUE_CONNECTION_STRING: "$(SB_CONN_STRING);EntityPath=processed-envelopes"
     PAYMENTS_QUEUE_CONNECTION_STRING: "$(SB_CONN_STRING);EntityPath=payments"
-    QUEUE_READ_INTERVAL: "30000"
-    DELETE_ENVELOPES_DLQ_MESSAGES_ENABLED: "true"
-    DELETE_ENVELOPES_DLQ_MESSAGES_CRON: "0 * * * * *"
-    DELETE_ENVELOPES_DLQ_MESSAGES_TTL: "10s"
-    TRANSFORMATION_URL_BULKSCAN: "http://bulk-scan-sample-app-aat.service.core-compute-aat.internal"
+  keyVaults:
+    "bulk-scan":
```

Comment:
do we need quotes?

Reply:
I don't think we need it can try removing it and see if it works.